internal/aws/datazone/environment_profile_resource_gen.go

```diff
@@ -334,6 +335,16 @@ func environmentProfileResource(ctx context.Context) (resource.Resource, error)
 
 	opts = opts.WithCloudFormationTypeName("AWS::DataZone::EnvironmentProfile").WithTerraformTypeName("awscc_datazone_environment_profile")
 	opts = opts.WithTerraformSchema(schema)
+	opts = opts.WithPrimaryIdentifier(
```

Comment:
This resource also needs an Account ID and Region to globally uniquely identify it